packages/yuudachi/src/commands/moderation/ban.ts

```diff
@@ -69,8 +69,6 @@ export default class extends Command<typeof BanCommand> {
 		const banKey = nanoid();
 		const cancelKey = nanoid();
 
-		const embed = await generateHistory(interaction, args.user, locale);
```

Comment:
Why those changes in every file? Seems pretty unnceessary.

Reply:
Will be reverted with the `generatedHistory` revert

Reply:
History reverted in 3646e40
Files changes improved in cc16e8f